app.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
- Can we change `name` to `SAML Jackson`
- For the `addons`, do you think it a good idea to keep it as `hobby-dev`?

Reply:
`hobby-dev` seems like a good default (free plan), I presume this can be changed by the user before deploying @niwsa?

Reply:
Don't see an option for setting the plan before deploying. After deploying on a `hobby-dev` plan [updating-with-pg-copy](https://devcenter.heroku.com/articles/updating-heroku-postgres-databases#updating-with-pg-copy) seems to to be the only way to change the plan.

Reply:
Removed the `hobby-dev` plan from `app.json`